### DISCUSSION TITLE
Connection automatique

### DISCUSSION THREAD

**Discussion by Jaffaden** [witsa/synapps#610] [Q&A] [answered]
Bonjour,
Je voudrais savoir si il est possible de connecter automatiquement l'utilisateur à un second automate Wit avec les logs utilisé sur le premier.

J'ai un automate qui gère ma chaufferie et qui à sont propre synapps, ainsi qu'un second automate qui va gérer un autre bâtiment avec des départs de chauffage. Les deux communiquent en IP j'ai donc fait un lien entre les deux synapps (pas encore tester car je n'ai pas les automates sous la main) mais lors du basculement entre le synapps de ma chaufferie vers celui des départs de chauffage il faudra que l'utilisateur ce re connecte. donc est-il possible de connecter automatiquement l'utilisateur (même via des logs faits pour) lorsque je clique sur mon lien ?

**Comment by WITNicolas** [comment, marked as answer]
Bonjour,

Malheuresement non, il n'y  a pas de connexion automatique, tout cela pour des questions de cybersecurité. Une pratique existait avec les anciens synoptiques (wlog), mais qui était justement une entorce à la cyber.
Pour les synapp, à la premiere connexion, il y a une case à cocher "Se souvenir de moi".
Il faut que dans chaque REDY les utilisateurs soient renseignés dans les programmes.

Il n'existe pas actuellement dans les REDY un serveur d'authentification type SSO ou LDAP.

Pour les liaisons entre synapp vous pouvez utiliser ce type d'url dans vos iFrame :
http://**_IPREDY_**/web/ria/synapps/latest/index.html?lang=fr#/**_LABELSYNAP_**

Cordialement,

**Reply by CvcRegulation-Jordan** [reply]
Salut Nicolas,
Je remonte sur ce vieux sujet ; Est-il possible dans une iFrame, d'appeler une scène spécifique d'un SynApps ? Si oui comment ?
A+

**Reply by WITNicolas** [reply]
Salut,

Tout betement en ajoutant la scene dans l'URL :)
http://IPREDY/web/ria/synapps/latest/index.html?lang=fr#/LABELSYNAP/LABELSCENE
